src/Renderer/ThreeExtended/GLTFLoader.js

```diff
@@ -1726,8 +1726,15 @@ THREE.GLTFLoader = ( function () {
 							meshNode = new THREE.LineSegments( geometry, material );
 
 						} else {
-
-							meshNode = new THREE.Line( geometry, material );
+							if (primitive.mode === WEBGL_CONSTANTS.LINES ) {
+								meshNode = new THREE.LineSegments( geometry, material );
+							} else if (primitive.mode === WEBGL_CONSTANTS.LINE_STRIP ) {
+								meshNode = new THREE.Line( geometry, material );
+							} else if (primitive.mode === WEBGL_CONSTANTS.LINE_LOOP) {
+								meshNode = new THREE.LineLoop( geometry, material );
+							} else {
+								meshNode = new THREE.Line( geometry, material );
```

Comment:
Shouldn't you throw an error instead if the mode is unknown ?

Reply:
We only enter here if:
```js
if ( primitive.mode === WEBGL_CONSTANTS.LINES
+							|| primitive.mode === WEBGL_CONSTANTS.LINE_STRIP
+							|| primitive.mode === WEBGL_CONSTANTS.LINE_LOOP )
```
is true anyway. If you want I can remove the else completely?

Reply:
You could remove the last else and do this:

```js
else { // WEBGL_CONSTANTS.LINE_LOOP)
    meshNode = new THREE.LineLoop( geometry, material );
}
```

Reply:
Updated.